test.js

```diff
@@ -26,6 +26,7 @@ it('should NOT find event in origin when it does not exist', () => {
 })
 
 it('should find event in destination when it exists', () => {
+  objectUnderTest.TEST_INCLUDE_DESC = true
```

Comment:
:pray: thanks for covering my mistake

Reply:
hah, it only happened when  I put in a suggestion via Github UI without testing it first :D